ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_MULTI_RR_TESTS.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Inherit most of the variables from CV_PATHFINDERS_TESTS
+
+wan_route_servers:
+  # Not testing the overloading
+  - hostname: cv-pathfinder-pathfinder1
+  - hostname: cv-pathfinder-pathfinder2
+  # This one is not in the inventory pathfinder1 & 2
+  - hostname: cv-pathfinder-pathfinder3
+    router_id: 6.6.6.6
+    wan_path_groups:
```

Comment:
since we are already under wan_route_servers, maybe it is enough with just `path_groups` ? The default is all path groups right?

Reply:
there is no default